src/app/features/create/components/hydrate-new-vehicle-record/hydrate-new-vehicle-record.component.html

```diff
@@ -0,0 +1,18 @@
+<ng-container *ngIf="vehicle$ | async as vehicle">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+          <app-tech-record-title [vehicleTechRecord]="vehicle || undefined" [hideActions]="true"></app-tech-record-title>
+          <app-button id="submit-record-continue" (clicked)="handleSubmit()">Submit new record</app-button>
+        </div>
+      </div>
+    </div>
+
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-grid-column-full">
+        <app-tech-record-summary [isEditing]="true" [vehicleTechRecord]="vehicle.techRecord[0]"></app-tech-record-summary>
```

Comment:
Realise this isn't what you've written but after not looking at VTM for a while I had to double check for my own sanity... but I think that vehicleTechRecord is badly named for what it stores, I'd have presumed it stored the whole vehicle object, and not just a technical record object...